### PR TITLE
Fetch and display hardcoded comments from server

### PR DESCRIPTION
**Pull request by jieliangang**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -24,9 +27,29 @@
...
 public class DataServlet extends HttpServlet {
 
+  private List<String> comments;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Jie Liang!</h1>");
+    
+    comments = new ArrayList<>();
```
Comment (henryken) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:35]: These comments seem a constant. Just as a discussion, are there other places that we can put these at?

Comment (jieliangang) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:35]: I could initialize with the list of comments?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -24,9 +27,42 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> comments = new ArrayList<>();;
```
Comment (henryken) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Remove extra semicolon

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (henryken) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: I see that you've moved it as class attribute.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -24,9 +27,42 @@
...
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment = getComment(request);
+    if(isValidComment(comment)) {
```
Comment (henryken) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:44]: What if the comment is not valid?

Comment (jieliangang) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:44]: Currently nothing is done. Server could send 400 bad request error? and client would have to handle the error accordingly.

Else, client side form validation can be done too.

Comment (henryken) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:44]: You can choose which one you prefer. Just asking how would you handle invalid comment.

**Pull request merged**